HerPublicWebsite/Models/Questionnaire/AddressViewModel.cs

```diff
@@ -1,7 +1,12 @@
+using GovUkDesignSystem.Attributes.ValidationAttributes;
+
 namespace HerPublicWebsite.Models.Questionnaire;
 
 public class AddressViewModel: QuestionFlowViewModel
 {
+    [GovUkValidateRequired(ErrorMessageIfMissing = "Enter a building number or name")]
```

Comment:
@ZacharyFreed I missed this when reviewing your changes. These fields should be mandatory, and this is how we can do that with the GovUK library.